tests/context.py

```
@@ -380,6 +380,108 @@
+			>>>
+			>>> test_fixture = ['COVERAGE', 'run']
+			>>> _context.taint_command_args(test_fixture)  #doctest: +ELLIPSIS
+			[...'COVERAGE', 'run']
+			>>>
+	"""
+	if not args or not isinstance(args, (list, tuple)):
+		raise CommandExecutionError("Invalid command arguments", exit_code=66)
+	# Validate the command (first argument)
+	allowed_commands = {
+		"python", "python3", "coverage", "coverage3",
+		sys.executable  # Allow the current Python interpreter
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)